management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -234,15 +235,22 @@ private void remotelyUpdateCollectedStatistics() throws Exception {
       assertThat(client, is(notNullValue()));
       final ClientIdentifier ehcacheClientIdentifier = client;
 
+      CountDownLatch callCompleted = new CountDownLatch(1);
       AtomicReference<String> managementCallId = new AtomicReference<>();
       BlockingQueue<ContextualReturn<?>> returns = new LinkedBlockingQueue<>();
 
       agent.setContextualReturnListener(new ContextualReturnListener() {
         @Override
         public void onContextualReturn(ClientIdentifier from, String id, ContextualReturn<?> aReturn) {
-          assertEquals(ehcacheClientIdentifier, from);
-          assertEquals(managementCallId.get(), id);
-          returns.offer(aReturn);
+          try {
```

Comment:
Does this really need to catch InterruptedException? I don't recall seeing this type of construct in 4.x tests that use a latch.